contracts/proposals/ListOnOpenseaProposal.sol

```diff
@@ -255,7 +270,14 @@ abstract contract ListOnOpenseaProposal is ZoraHelpers {
             ? IOpenseaExchange.OrderType.FULL_OPEN
             : IOpenseaExchange.OrderType.FULL_RESTRICTED;
         orderParams.salt = uint256(bytes32(domainHashPrefix));
-        orderParams.conduitKey = conduitKey;
+        {
+            // Approve OpenSea's conduit to spend our NFT. This should revert if we
+            // do not own the NFT.
+            bytes32 conduitKey = _GLOBALS.getBytes32(LibGlobals.GLOBAL_OPENSEA_CONDUIT_KEY);
+            (address conduit, ) = CONDUIT_CONTROLLER.getConduit(conduitKey);
+            token.approve(conduit, tokenId);
+            orderParams.conduitKey = conduitKey;
+        }
```

Comment:
awkward placement for a side-effect imo, maybe move to L315 and you can just work on `orderParams.conduitKey` directly?